src/main/resources/errors.yaml

```diff
@@ -612,6 +612,18 @@ request-errors:
       
       Resend the command using only supported column types.
 
+  - scope: PROJECTION
+    code: UNSUPPORTED_AMPERSAND_ESCAPE_USAGE
+    title: The usage of ampersand escape is not supported
+    body: |-
+      The usage of ampersand escape is not supported.
+      
+      Ampersand `&` escape is only allowed to escape and followed by ampersand `&` and dot `.`.
```

Comment:
-> "Ampersand `&` can only be used as an escape character and must be followed by a `&` or `.` character"